client/app/ui-components/tile/tile.component.js

```diff
@@ -135,44 +135,40 @@ angular.module("managerApp")
         scope: true,
         template: `
```

Comment:
Can this be extracted into a file ? 😃

Reply:
No. It has been built like that. We'll put an effort on it when we'll turn into oui-tile ;)